pagination/src/test/scala/com/cerner/beadledom/pagination/OffsetPaginatedListLinksWriterInterceptorSpec.scala

```diff
@@ -71,7 +71,7 @@ class OffsetPaginatedListLinksWriterInterceptorSpec extends AnyFunSpec with Matc
     val queryString = queryParams
         .filter({ case (_, v) => v != null })
         .map({ case (k, v) => s"$k=$v" }).mkString("&")
-    val uriInfo = new ResteasyUriInfo("example.com", queryString, "")
+    val uriInfo = new ResteasyUriInfo("example.com", queryString, null)
```

Comment:
You should be able to use `ResteasyUriInfo(final String absoluteUri, final String contextPath)` constructor.

Reply:
f2008b27